tests/github-glue.test.ts

```diff
@@ -63,9 +63,6 @@ test("identify user", async () => {
         const github = new GitHubProxy(`../${repo}`, repo);
         await github.authenticate(owner);
 
-        const name = await github.getGitHubUserName(owner);
-        expect(name).toMatch(userName);
-
         const ghUser = await github.getGitHubUserInfo(owner);
         expect(ghUser.login).toMatch(owner);
         expect(ghUser.name).toMatch(userName);
```

Comment:
@dscho Should this test allow `ghUser.name` to be empty?  ie, should GGG contributors be allowed to have no name or should this be a reminder that a name is needed?  Not related to this change and no reason to hold it up but it comes to mind after the previous change.

Only testers with a test repo will actually run this test so it may not show up anyway. @ugultopu Did you set up a test repo when testing?  The basic instructions are at the start of `github-glue.test.ts`.  Also interested in feedback if the instructions are not clear.

Reply:
No, I didn't set up a test repo, just realized that `getGitHubUserName` wasn't referenced (anymore?) in the TypeScript files and hence, removed it. So `getGitHubUserName` is still being used somewhere?

Reply:
> `getGitHubUserName` wasn't referenced (anymore?)

The last reference may have been removed in PR #110, which would make this something I should have removed.

Reply:
> should GGG contributors be allowed to have no name

A `/submit` will fail if no name was provided.